### ISSUE TITLE
attack type in response on the 403 page

### ISSUE THREAD

**Issue by meisamrce** [corazawaf/coraza#819]
Hi 
How can the attack type be displayed to the user as a response on the 403 page?

for example 

http://localhost/?id=1+union+select+           # 403 response send message attack detection SQL Injection

**Comment by M4tteoP** [comment]
Hi @meisamrce, thanks for reaching out.

While I think it is generally not advisable to provide any details about an interruption that occurred, I see that it might come in handy for testing purposes.

The connector is the component in charge of enforcing the interruption and generating the denial response. I would see it as a feature request connector-side. Which connector are you using?

Furthermore, running the `CRS` or any ruleset with a similar logic to the `CRS anomaly score`, an interruption might come from multiple triggered rules, it would lead to displaying a list of triggered rules (or their message/category). Basically, I guess you are looking for something similar to the [CRS sandbox output](https://coreruleset.org/docs/development/sandbox/) (Note Coraza support in the CRS sandbox is WIP), but directly built into the Coraza connector(s) 🤔

**Comment by meisamrce** [comment]
I have used your web server for example
And I want to have the type of attack in the response 

In any way possible, I would be grateful if you could put a simple example and how to get the type of attack in the server's response


thank you

**Comment by jcchavezs** [comment]
Are you willing to get the rule that was triggered in a response header for
example?

**Comment by meisamrce** [comment]
Yes, I want to find out what type of attack it was, and in response, it doesn't matter in what form, and I don't want to check the log file.

**Comment by jcchavezs** [comment]
I think we could have an option in the http handler to return the matched rules in a response handler and also the transaction ID.
